### ISSUE TITLE
How to fix ZoneId, when I use ZonedDateTime

### ISSUE THREAD

**Issue by veluxer62** [appmattus/kotlinfixture#65]
Is there any configuration to fasten `ZoneId` when using `ZonedDateTime`?

**Comment by mattmook** [comment]
Currently no. Will see what I can do to make it possible, basically the ZoneId should be resolved but it's always picking one currently

**Comment by mattmook** [comment]
Released in v0.9.6

To override:

```
val fixture = kotlinFixture {
    factory<ZoneId> { ZoneId.of("Europe/London") }
}

fixture<ZonedDateTime>()
```

or

```
val fixture = kotlinFixture()

fixture<ZonedDateTime> {
    factory<ZoneId> { ZoneId.of("Europe/London") }
}
```

**Comment by veluxer62** [comment]
@mattmook Thanks!!
